client/curriculum/html/Lists.tsx

```diff
@@ -12,6 +12,7 @@ import Lesson, {
 import { Diana } from '~/services/contributors';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.